dapr-http-client/src/Dapr/Client/HttpClient/Metadata.hs

```diff
@@ -16,6 +16,5 @@ getMetadata config = do
 addCustomLabel :: MonadIO m => DaprConfig -> Text -> Text -> m (Either DaprClientError ())
 addCustomLabel config attribute rawData = do
   let url = ["metadata", attribute]
-      options = header "Content-Type" "text/plain"
-  response <- makeHttpRequest config PUT url (ReqBodyBs $ encodeUtf8 rawData) ignoreResponse options
+  response <- makeHttpRequest config PUT url (ReqBodyBs $ encodeUtf8 rawData) ignoreResponse mempty
```

Comment:
`(header "Content-Type" "text/plain")` shouldn't be removed since it's text message instead of JSON.